model/installation_database.go

```diff
@@ -87,6 +87,7 @@ type InstallationDatabaseStoreInterface interface {
 	GetProxyDatabaseResourcesForInstallation(installationID string) (*DatabaseResourceGrouping, error)
 	GetOrCreateProxyDatabaseResourcesForInstallation(installationID, multitenantDatabaseID string) (*DatabaseResourceGrouping, error)
 	DeleteInstallationProxyDatabaseResources(multitenantDatabase *MultitenantDatabase, databaseSchema *DatabaseSchema) error
+	GetGroupDTOs(filter *GroupFilter) ([]*GroupDTO, error)
```

Comment:
I'm not enjoying this interface's name. Would like suggestions. `ProvisionerStore` ?

Reply:
Are you going to add implementation of `GetGroupDTOs` later?

Reply:
Implementation already exists in [internal/store/group_dto.go](https://github.com/mattermost/mattermost-cloud/blob/master/internal/store/group_dto.go#L30)